core/common/src/ConcurrentQuery.ts

```diff
@@ -61,8 +61,12 @@ export interface QueryPropertyMetaData {
   jsonName: string;
   /** The name is the property's alias if the property is a generated one, otherwise, it is the name of the property. */
   name: string;
-  /** If this property is a PrimitiveECProperty, extend type is the extended type name of this property, if it is not defined locally will be inherited from base property if one exists, otherwise extended type is set to an empty string. */
+  /** If this property is a PrimitiveECProperty, extend type is the extended type name of this property, if it is not defined locally will be inherited from base property if one exists, otherwise extend type is set to an empty string.
+   * @deprecated in 4.10 Use extendedType instead
```

Comment:
I have written @deprecated in 4.10    Is the version number okay? Or should I write something else?
@khanaffan

Reply:
4.10 is already released. It'll have to go in 4.11

Reply:
Changed the version to 4.11